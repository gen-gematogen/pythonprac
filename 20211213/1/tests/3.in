[22, 58, 95, -332, 47, 131, 6276, 38, 70, 184, -1140, 35, 56, 28, -13, -1]
