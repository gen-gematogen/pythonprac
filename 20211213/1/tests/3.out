[-1140, -332, -13, -1, 22, 28, 35, 38, 47, 56, 58, 70, 95, 131, 184, 6276]
